dev_guide/builds.adoc

```diff
@@ -17,6 +17,74 @@ runnable images to be used on OpenShift. There are three types of builds:
 - STI build
```

Comment:
and I just realized we have this thing already....
http://docs.openshift.org/latest/architecture/core_objects/builds.html#buildconfig